docs/deploy/ansible/k3s.md

```diff
@@ -0,0 +1,52 @@
+# K3S
+
+*Playbook Path: [ansible/playbooks/install_k3s.yml](https://github.com/webrecorder/browsertrix-cloud/blob/main/ansible/playbooks/install_k3s.yml)*
+
+This playbook provides an easy way to install Browsertrix Cloud on a Linux box (tested on Rocky Linux 9) 
+It automatically sets up Browsertrix with, Letsencrypt certificates.
+
+### Requirements
+
+To run this ansible playbook, you need to:
+
+* Have a server / VPS where browsertrix will run.
+* Configure a DNS A Record to point at your server's IP address.
+* Make sure you can ssh to it, with a sudo user: ssh <your-user>@<your-domain>
+* Install Ansible on your local machine (the control machine).
+
+
+1. Clone the repo:
+```zsh
+git clone https://github.com/webrecorder/browsertrix-cloud.git
+cd browsertrix-cloud
+```
+
+2. Create a copy of the [inventory directory] and name it what you like:
+```zsh
+cp -r ansible/inventory/sample-k3s ansible/inventory/my-deployment
+```
+
+1. [Look at the configuration options](https://github.com/webrecorder/browsertrix-cloud/blob/main/ansible/inventory/sample-k3s/group_vars/all.yml) and modify them to match your setup 
+
+2. Change the [hosts IP address](https://github.com/webrecorder/browsertrix-cloud/blob/main/ansible/inventory/sample-k3s/hosts.ini) in your just created inventory
+
+3. Copy the playbook into the root ansible directory:
+```zsh
+cp ansible/playbooks/install_k3s.yml ansible/install_k3s.yml
```

Comment:
is copying needed in case user wants to modify playbook, otherwise can it work as is?